[feat](oss)Support datalake VPC Endpoint Format (dlf / datalake-vpc)

CalvinKirs opened the pull request:
FYI  https://help.aliyun.com/zh/dlf/dlf-1-0/developer-reference/api-datalake-2020-07-10-endpoint

CalvinKirs committed:
[feat](oss)Support DeltaLake VPC Endpoint Format (dlf / deltalake-vpc)

FYI  https://help.aliyun.com/zh/dlf/dlf-1-0/developer-reference/api-datalake-2020-07-10-endpoint

Thearas commented:
Thank you for your contribution to Apache Doris.
Don't know what should be done next? See [How to process your PR](https://cwiki.apache.org/confluence/display/DORIS/How+to+process+your+PR).

Please clearly describe your PR:
1. What problem was fixed (it's best to include specific error reporting information). How it was fixed.
2. Which behaviors were modified. What was the previous behavior, what is it now, why was it modified, and what possible impacts might there be.
3. What features were added. Why was this function added?
4. Which code was refactored and why was this part of the code refactored?
5. Which functions were optimized and what is the difference before and after the optimization?

CalvinKirs committed:
[feat](oss)Support DeltaLake VPC Endpoint Format (dlf / deltalake-vpc)

FYI  https://help.aliyun.com/zh/dlf/dlf-1-0/developer-reference/api-datalake-2020-07-10-endpoint

hello-stephen commented:
# FE UT Coverage Report
Increment line coverage `  100.00% (2/2)` :tada:
[Increment coverage report](http://coverage.selectdb-in.cc/coverage/56473_5be7e8c58d06895262d2c78a8e71d1e0cc42ae7a/fe_increment_report/index.html)
[Complete coverage report](http://coverage.selectdb-in.cc/coverage/56473_5be7e8c58d06895262d2c78a8e71d1e0cc42ae7a/fe_report/index.html)
